### ISSUE TITLE
In Django, "Unknown command: 'raven'"

### ISSUE THREAD

**Issue by glarrain** [getsentry/raven-python#359]
According to http://raven.readthedocs.org/en/latest/config/django.html, "Once you’ve configured the client, you can test it using the standard Django management interface". This was added in fb4181433d3613c647f288d364a5697eae7152e9.

With Raven 3.4.1 and Django 1.4.8:
```
$ python manage.py raven test
Unknown command: 'raven'
Type 'manage.py help' for usage.
```

**Comment by xordoquy** [comment]
I'm closing this because I haven't been able to reproduce it. I'd be happy whenever someone can setup up an environment where this happens and which I can investigate.

**Comment by chris-erickson** [comment]
I'm having a similar problem right now.  At this point, it seems to be an issue with the DSN not being set, but as far I know right now, it's not a raven issue - setting up raven depends on a `dsn` var being set.  I use `django-getenv`to load vars like: `dsn = env("RAVEN_DSN", None)` and in my case, dsn isn't set.  I haven't figured out why yet.

**Comment by chris-erickson** [comment]
I eventually found the problem, and it had to do with the git commit integration.  Originally, I was using:

```python
'release': raven.fetch_git_sha(os.path.dirname(__file__)),
```
which worked for months.  It was unable to find a git repo at that location any longer.  It doesn't exist at the same directory as `settings.py` but git commands used to (and should) work anywhere below the git repo directory.  To resolve this, I just changed to:

```python
'release': raven.fetch_git_sha(DJANGO_ROOT),
```
and we're back in business.
